deploy/kubernetes-manifests.yaml

```diff
@@ -0,0 +1,185 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: smi-adapter-istio
+  namespace: istio-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: smi-adapter-istio
+  template:
+    metadata:
+      labels:
+        name: smi-adapter-istio
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      serviceAccountName: smi-adapter-istio
+      containers:
+        - name: smi-adapter-istio
+          image: {{ .Values.image.name }}
```

Comment:
This should be `deislabs/smi-adapter-istio:latest`, as we are using a single delpoyment file, right?

Reply:
fixed thank you!